deepee/wrapper.py

```diff
@@ -252,17 +250,7 @@ def prepare_next_batch(
         self._steps_taken += 1
         self._forward_succesful = self._clip_succesful = self._noise_succesful = False
         if self.watchdog:
-            self.watchdog.inform(self._steps_taken)
-
-        if not self.watchdog and return_privacy_spent:
-            raise RuntimeError(
-                "No PrivacyWatchdog is attached to the model. To return "
-                " privacy spent, please attach a watchdog."
-            )
-        elif self.watchdog and return_privacy_spent:
-            return self._privacy_spent
-        else:
-            return None  # just for you MyPy...
+            self._privacy_spent = self.watchdog.inform(self._steps_taken)
```

Comment:
inform now has a None return type. Shouldn't this now be `self._privacy_spent = self.current_epsilon(self._steps_taken)` ?